scylla-cql/src/types/serialize/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod serialize_row;
+pub mod serialize_value;
```

Comment:
The `serialize_` part seems unnecessary, these already are in a module called `serialize`.